src/main/java/com/serotonin/bacnet4j/test/DaqTest/Main.java

```diff
@@ -0,0 +1,25 @@
+package com.serotonin.bacnet4j.test.DaqTest;
+
+public class Main {
+
+  public static void main(String[] args) throws Exception {
+
+    if (args.length < 3) {
+      throw new IllegalArgumentException("Usage: bacnetTestId, broadcastIp, localIp");
```

Comment:
typically this would be something like:

Usage: <backnetTestId> <broadcastIp> <localIp>

Reply:
Sorry, I didn't get this comment. Is the string wrong?

Reply:
Sorry, I didn't get this comment. What the string should be like?

Reply:
Oh werid, my comment got cut off. It should have read:

Usage: <bacnetTestId> <broadcastIp> <loaclIp> 

basically no commas is the main important thing.